mrmap/monitoring/models.py

```diff
@@ -114,21 +125,15 @@ def get_add_action(cls):
                           needs_perm=PermissionEnum.CAN_ADD_MONITORING_RUN.value)
 
     def get_absolute_url(self):
-        return f"{reverse('monitoring:run_overview')}?uuid={self.uuid}"
+        return f"{reverse('monitoring:run_overview')}?id={self.id}"
```

Comment:
inherit from GenericModelMixin and fix the url namings to get url building with our common solution.

Reply:
Ok